release_notes.md

```diff
@@ -1,11 +1,23 @@
 # What's New
 
-- This solution pack is now **Uncertified**
+<table>
+    <tr>
+        <td><strong>NOTE</strong></td>
+        <td>This solution pack is now Certified.</td>
+    </tr>
+</table>
+
+## Playbook Enhancements
+
+- Updated the *Create Data Compliance Record* playbook to support data compliance record creation for both HIPAA and GDPR.
 
 ## Module Enhancements 
 
-We have made changes in the module so it can be used by the both HIPAA and GDPR Framework solution packs. To support a situation when both HIPAA and GDPR solution packs are deployed on the instance, we are aligning the data compliance module in both of these solution packs to avoid any conflict during the deployment of these solution packs.
-#### 1. Data Compliance Module
+- Enhanced the **Data Compliance** module to be used by the both HIPAA and GDPR Framework solution packs.
+    - For situations where both HIPAA and GDPR solution packs are deployed on the instance, the enhanced **Data Compliance** module avoids conflicts during the deployment of these solution packs.
+
+### Data Compliance Module
+
 Newly Added Fields - Specifically for HIPAA Framework Solution pack
```

Comment:
@cs-abhishek-shukla 
Please remove the below fields from the list

- Is A Breach Reportable
- What Health Information Was Exposed
- Reminder Mail Status

Reply:
Fixed @cs-rohit-khune